projects/plugins/jetpack/modules/shortcodes/vimeo.php

```diff
@@ -246,53 +255,6 @@ class_exists( 'Jetpack_AMP_Support' )
 }
 add_shortcode( 'vimeo', 'vimeo_shortcode' );
 
-/**
- * Callback to modify output of embedded Vimeo video using Jetpack's shortcode.
- *
- * @since 3.9
- *
- * @param array $matches Regex partial matches against the URL passed.
- * @param array $attr    Attributes received in embed response.
- * @param array $url     Requested URL to be embedded.
- *
- * @return string Return output of Vimeo shortcode with the proper markup.
- */
-function wpcom_vimeo_embed_url( $matches, $attr, $url ) {
```

Comment:
Instead of just removing the function, I would recommend marking it as deprecated first, and only remove it in a few releases. Since Vimeo embeds have been around for a long time, it's possible that third-parties currently do something with those functions.

Same for other functions in this PR.

Reply:
Updated!